subsys/bluetooth/host/smp.c

```diff
@@ -276,6 +276,7 @@ static bool oobd_present;
 static bool sc_supported;
 static const uint8_t *sc_public_key;
 static K_SEM_DEFINE(sc_local_pkey_ready, 0, 1);
+static bool duplicate_rl;
```

Comment:
Shouldn’t this be part of the SMP context struct instead of being a global variable?  We support multiple connections and in theory you could have multiple simultaneous SMP sessions.